### ISSUE TITLE
Add support for zkEVM

### ISSUE THREAD

**Issue by geleeroyale** [Giveth/giveconomy-balance-aggregator#6]
Cardona testnet and zkEVM mainnet

**Comment by mohammadranjbarz** [comment]
@geleeroyale 
The assignee of this should be @mhmdksh

**Comment by mhmdksh** [comment]
@geleeroyale @mohammadranjbarz I see that the configs are already in place for zkEVM on the staging and on production. And the subgraphs for them are already deployed. Is there anything else required for this setup??

**Comment by mhmdksh** [comment]
Staging:
<img width="658" alt="image" src="https://github.com/user-attachments/assets/d2bcc084-2f1a-4da9-9b51-7b183cd6a5af">

Prod:
<img width="664" alt="image" src="https://github.com/user-attachments/assets/4b8ad2f0-a9f3-4f95-a678-4d83eb904c9d">

**Comment by mhmdksh** [comment]
@laurenluz I think this has been done already and deployed. Let us know if all is well on your end

**Comment by laurenluz** [comment]
@mohammadranjbarz are we all good here?

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz are we all good here?

Yes it's implemented, we just need to wait until the end of the rounds to @maryjaf confirms it

**Comment by laurenluz** [comment]
@maryjaf can we close this issue now? we should have finished the 1st round by now, not sure what you needed to test.
